test/unit/lib/api-client.js

```diff
@@ -618,14 +663,29 @@ describe('API Client', () => {
             endpoint: 'checkout/longterm',
             json: {
                 studentId: 123456,
-                equipmentAddresses: ['iGwEZUvfA', 'iGwEZVHHE'],
+                equipment: [
+                    {
+                        address: 'iGwEZUvfA'
+                    },
+                    {
+                        address: 'iGwEZVHHE'
+                    }
+                ],
                 dueDate: today.toDateString(),
                 professor: 'professor',
                 adminCode: 123456
             },
             response
         });
-        return checkOutContentsLongterm(123456, ['iGwEZUvfA', 'iGwEZVHHE'], today.toDateString(), 'professor', 123456).then(data => {
+        let equipment = [
```

Comment:
Nit: you could really just move this statement up more to use it in `mockServer.expect` too